base-theme/assets/css/variables.scss

```diff
@@ -72,23 +73,53 @@ $font-normal: 1rem; /* 16px */
 $font-sm: 0.875rem; /* 14px */
 $font-xs: 0.75rem; /* 12px */
 
-// calculated pixel values for headings below are for courses theme
-// to calculate them for www theme, multiply rem value with 16px
 // h1
-$heading-one-font-lg: 2.28rem; /* 32px */
-$heading-one-font-xs: 1.71rem; /* 24px */
+$heading-one-font-lg: 2rem; /* 32px */
+$heading-one-font-xs: 1.5rem; /* 24px */
+
+$heading-one-line-height-lg: 2.25rem; /* 36px */
+$heading-one-line-height-xs: 1.75rem; /* 28px */
+
+$course-heading-one-font-lg: 2.28rem; /* 32px */
+$course-heading-one-font-xs: 1.71rem; /* 24px */
+
+$course-heading-one-line-height-lg: 2.57rem; /* 36px */
+$course-heading-one-line-height-xs: 2rem; /* 28px */
+
 // h2
-$heading-two-font-lg: 1.6rem; /* 22.4px */
-$heading-two-font-xs: 1.43rem; /* 20px */
+$heading-two-font-lg: 1.4rem; /* 22.4px */
+$heading-two-font-xs: 1.25rem; /* 20px */
+
+$heading-two-line-height-lg: 1.6rem; /* 25.6px */
+$heading-two-line-height-xs: 1.5rem; /* 24px */
+
+$course-heading-two-font-lg: 1.6rem; /* 22.4px */
+$course-heading-two-font-xs: 1.43rem; /* 20px */
+
+$course-heading-two-line-height-lg: 1.83rem; /* 25.6px */
+$course-heading-two-line-height-xs: 1.71rem; /* 24px */
+
 // h3
```

Comment:
Don't need (xs) and (lg) if they have same value. A single variable should suffice; Removing breakpoints for such cases to reduce code redundancy.